actions/release/reset-draft/entrypoint/main_test.go

```diff
@@ -364,6 +448,9 @@ func TestEntrypoint(t *testing.T) {
 					Expect(buffer).To(gbytes.Say(`{"error": "server-error"}`))
 				})
 			})
+			context.Focus("when a specific version is passed in", func() {
```

Comment:
I think we should remove the `Focus` and it seems like we should either remove this empty context or add some tests to it.

Reply:
yes absolutely